samples/bluetooth/mesh_provisioner/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _ble_mesh_provisioner:
+
+Bluetooth: Mesh Provisioner
+###########################
+
+Overview
+********
+
+TBD
+
+Requirements
+************
+
+TBD
+
+Building and Running
+********************
+
+TBD
```

Comment:
Do you have plans to fill these in before the next release (i.e. in the coming weeks)?

Reply:
@jhedberg I just updated it. Please review :)